draft-ietf-tls-ctls.md

```diff
@@ -808,27 +812,29 @@ This document requests that IANA change the name of entry 6 in the TLS
 HandshakeType Registry from "hello_retry_request_RESERVED" to
 "hello_retry_request", and set its Reference field to this document.
 
-## Reserved profiles
+## cTLS Profile ID Registry {#reserved-profiles}
 
-This document requests that IANA open a new registry entitled "Well-known
-cTLS Profile IDs", on the Transport Layer Security (TLS) Parameters page,
+This document requests that IANA open a new registry entitled
+"cTLS Profile IDs", on the Transport Layer Security (TLS) Parameters page,
 with the following columns:
 
-* ID value: A sequence of 1-4 octets.
-* Template: A JSON object.
-* Note: An explanation or reference.
+* Value: A sequence of 1-4 octets.
+* Name: Name of the profile.
+* Description: Short description of the profile.
+* Reference: Reference to the document or documents that specify the profile.
+
+The ID values of length 1 are subject to a "RFC Required" registry
+policy. Values of length 2 are subject to an "Specification Required" policy.
+Values of length 3 and 4 are subject to a "Private Use" policy.
```

Comment:
I don't understand the utility of a "private use" range here.  All IDs longer than 4 bytes are already basically "private use".  It would seem more sensible to me to shorten Value to 1-2 octets, or possibly to mark lengths 3 and 4 as Reserved.